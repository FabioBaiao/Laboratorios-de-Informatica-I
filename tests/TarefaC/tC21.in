Caabb
cbaDc
3 1 S
DSAESDSDALDSASADSDSL