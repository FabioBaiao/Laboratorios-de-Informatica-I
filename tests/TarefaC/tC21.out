0 1
3 0
FIM 20
